tests/linters_output/bandit-json.txt

```diff
@@ -0,0 +1,624 @@
+{
+  "errors": [],
+  "generated_at": "2021-01-07T23:39:39Z",
+  "metrics": {
+    "./build/lib/lintly/__init__.py": {
```

Comment:
Nit: including a few of these 'metrics' blocks is ok, but this is a little *too* spammy, especially considering we never use any of the values in here at all. Maybe remove all of them except for 1 or 2?

Reply:
Ok make sense.